soccer/planning/RRTPlanner.cpp

```diff
@@ -137,15 +139,24 @@ Planning::InterpolatedPath* RRTPlanner::run(
 	{
 		// FIXME: without these two lines, an empty path is returned which causes errors down the line.
 		path->points.push_back(start);
+		path->times.push_back(0);
+		path->vels.push_back(Geometry2d::Point(0,0));
 	}
 	return path;
 }
-
+Planning::InterpolatedPath update(
```

Comment:
Oh ok.  Wouldn't we just decide where we want to replan from, then pass that <pos, vel> pair to the planner?  What would `update()` do?